OAuth/ResourceOwner/AbstractResourceOwner.php

```diff
@@ -301,9 +301,9 @@ protected function generateNonce()
     /**
      * Configure the option resolver
      *
-     * @param OptionsResolverInterface $resolver
+     * @param OptionsResolver $resolver
      */
-    protected function configureOptions(OptionsResolverInterface $resolver)
+    protected function configureOptions(OptionsResolver $resolver)
```

Comment:
> If tests are passing with old version than that's ok

Tests works for both old and new signature (and both usage at the same time, see: https://github.com/hwi/HWIOAuthBundle/pull/773#discussion_r31893460).

So I think there is not problem for this.